docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -0,0 +1,248 @@
+MLflow Tracking Quickstart
+==========================
+
+Welcome to MLflow! 
+
+The purpose of this quickstart is to provide a quick guide to the most essential core APIs of MLflow Tracking. 
+Specifically, those that enable the logging, registering, and loading of a model for inference. 
+
+.. note::
+    For a more in-depth and tutorial-based approach (if that is your style), please see the 
+    `Getting Started with MLflow <../logging-first-model/index.html>`_ tutorial. We recommend that you start here first, though, as this quickstart 
+    uses the most common and frequently-used APIs for MLflow Tracking and serves as a good foundation for the other tutorials in the documentation. 
+
+What you will learn
+-------------------
+
+In just a few minutes of following along with this quickstart, you will learn:
+
+* How to **log** parameters, metrics, and a model
+* The basics of the **MLflow fluent API**
+* How to **register** a model during logging 
+* How to navigate to a model in the **MLflow UI** 
+* How to **load** a logged model for inference 
+
+If you would like to see this quickstart in a purely notebook format, we have a downloadable and viewable notebook-only version of this quickstart:
+
+.. raw:: html
+
+    <a href="notebooks/index.html" class="download-btn">View the Notebook</a>
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    notebooks/index
+
+
+Step 1 - Get MLflow
+-------------------
+
+MLflow is available on PyPI. If you don't already have it installed on your system, you can install it with:
+
+.. code-section::
+    .. code-block:: bash
+        :name: download-mlflow
+
+        pip install mlflow
+
+Step 2 - Start a Tracking Server
+--------------------------------
+
+We're going to start a local MLflow Tracking Server, which we will connect to for logging our data for this quickstart.
+From a terminal, run:
+
+.. code-section::
+    .. code-block:: bash
+        :name: tracking-server-start
+
+        mlflow server --host 127.0.0.1 --port 8080
+
+.. note::
+    You can choose any port that you would like, provided that it's not already in use. 
+
+Step 3 - Train a model and prepare metadata for logging
+-------------------------------------------------------
+
+In this section, we're going to log a model with MLflow. A quick overview of the steps are:
+
+- Load and prepare the Iris dataset for modeling.
+- Train a Logistic Regression model and evaluate its performance.
+- Prepare the model hyperparameters and calculate metrics for logging.
+
+
+.. code-section::
+    .. code-block:: python
+        :name: train-model
+
+        import mlflow
+        import mlflow.sklearn
+        from mlflow.models import infer_signature
+
+        import pandas as pd
```

Comment:
```suggestion
```

Reply:
We use this in step 5 in crafting the `result` DataFrame